### PR TITLE
Added DLSR using AutoEncoders

### LINKED ISSUE #1016: Title: Deep Learning Super Resolution on Any Image
### Is your feature request related to a problem? Please describe.

This issue addresses the issue of low-resolution or damaged images lacking detail and clarity. My project focuses on deep learning super resolution techniques to upscale images into high resolutions while preserving crucial visual details. The goal is to enhance image quality for various applications like image processing and computer vision.
This issue is faced by millions of people across the world and the highest use case is with old images and damaged images in natural disasters or accidents.

I would like to take this task under GSSOC'24.

Hope I make it to your sight..

Thanks.

### Describe the solution you'd like.

The solution involves implementing a deep learning-based super resolution model. This model will take low-resolution images as input and utilize advanced neural network architectures to upscale them into high-resolution outputs. By training the model on a diverse dataset of low and high-resolution image pairs, it will learn to capture and enhance important visual details, resulting in significantly improved image quality.
It will include PyTorch,CUDA ,etc along with 4 datasets to accurately train the model to perform the task.

### Describe alternatives you've considered.

There is currently no alternative to my proposed task.

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by Soubeer**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

In this project, I developed a deep learning model using autoencoders to enhance the resolution of low-resolution or damaged images. By training the autoencoder on a dataset of paired high-resolution and low-resolution images, the model learns to reconstruct high-quality images from degraded inputs. This technique leverages the power of neural networks to perform image super-resolution, effectively restoring and improving image quality, making it applicable for various uses such as photo restoration and enhancing visual content in low-bandwidth environments.

Fixes #1016 

### Checklist

- [✓ ] I've read the contribution guidelines.
- [ ✓] I've checked the issue list before deciding what to submit.
- [ ✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Nothing similar

### CODE REVIEW

**Comment by Soubeer** [comment]
@Kushal997-das Sir I am sorry , I was really unable to find my Previous PR and thought I didn't make one. I am new to these so I am still finding my way around the block.
If possible can you check one PR and close the other which I opened by mistake?
It would be really helpful.

Thank You

**Comment by Soubeer** [comment]
> @Soubeer Hello I found you want to add the same project in another repo which is against the rule so I'm closing the PR.
> 
> [here](https://github.com/Kushal997-das/Project-Guidance/pull/1113)

@Kushal997-das Sir the link which you attached for the duplicate repo is this same repo and the same PR page.. It does not open any other repo and as far as I can find, I only made one PR for this repo and one task on this topic for this PR.

**Comment by Kushal997-das** [comment]
By mistake, I pasted a different link, But it was there, you also said sorry, and now you deleted your comment, that seems how you are. See below attached image and your reply.

spam

**Comment by Soubeer** [comment]
> By mistake, I pasted a different link, But it was there, you also said sorry, and now you deleted your comment, that seems how you are. See below attached image and your reply.
> 
> spam

@Kushal997-das Yes I did deleted the comment because I would definitely trust you when you say that I made a duplicate repo.So I apologized before even checking.
Then when I went to check there was nothing.
You can check the logs or something from github and show me where the duplicate repo is.

I would not go this far cheating for 10 points. Its not worth it

**Comment by Kushal997-das** [comment]
I have seen it clearly, by mistake I put another link. I'll not accept your PR.

**Comment by Soubeer** [comment]
its okay you might not accept. But why are you accusing me of cheating here for "10 points" ? And now you cannot provide me the proof? 
This is not how things work in github ig? There is always a log of changes somewhere. 
I dont want to be rude but this is not acceptable from my side as well. I have checked other PRs also and they have been given the proof of duplicate task. 
I will accept my fault if you can give me mine.

I had 10 repo yesterday and I have the same right now too. I have mailed all my work to the GSSOC team yesterday regarding leaderboard so they have the proof of my work. I dont know how this all adds up to " that seems how you are".

**Comment by Kushal997-das** [comment]
Okey, I'll open it again make sure there shouldn't be any duplicate I don't
have much time to check thoroughly.

**Comment by Soubeer** [comment]
Thank you Sir, I have checked again now , there are no duplicate repos having the same task.
Kindly open this PR again and assign me the label if the task is complete.

**Comment by Soubeer** [comment]
@Kushal997-das can you open this and merge my file if its okay and provide me with a level?

**Comment by Kushal997-das** [comment]
@Soubeer Will check .

File: Machine Learning and Data Science/Intermediate/Deep_Learning_Super_Resolution/READme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Contributing
+Contributions to this project are welcome! Whether you have ideas for new features, improvements to existing code, or bug fixes, your contributions are valuable. To contribute:
+
+* Fork the repository: Create your own copy of the project on GitHub.
+* Create a new branch: Make a new branch for your feature or bugfix.
+* Make your changes: Implement your changes in this branch.
+* Commit your changes: Save your changes with a descriptive commit message.
+* Push to your branch: Upload your changes to your GitHub fork.
+* Open a pull request: Submit your changes for review and integration into the main project.
```
Comment (Kushal997-das) [inline Machine Learning and Data Science/Intermediate/Deep_Learning_Super_Resolution/READme.md:39]: Update the details according to project-guidance. In the readme.md file, include only information related to the project. There is no need to add extra information such as contributions and other details

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Soubeer, the following things need attention:

Add the project under the "basic" section.
Your .ipynb file is not opening.
Update the readme.md file.
Provide detailed instructions in the readme.md file.
Change the project name from Deep_Learning to a more logical name.

**Comment by Soubeer** [comment]
@Kushal997-das changes made

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Soubeer Hi, I'm not feeling well, so I'm taking some time to rest.

Could you please give the project a more logical name? 'Deep_learning' doesn't seem suitable. Try to come up with a name that better fits the nature of the project.

Also, make sure you fetch the updated version of the project.

**Comment by Soubeer** [comment]
@Kushal997-das Changed the name to Super_Resolution_Using_Deep_Learning

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Soubeer As mentioned in last comment fetch the repo once again.

**Comment by Soubeer** [comment]
@Kushal997-das updated all the necessities.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Hi @Soubeer, I can't keep telling you the same thing again and again. Yet, you still don't do things properly. [Here](https://github.com/Soubeer/Project-Guidance/blob/main/Machine%20Learning%20and%20Data%20Science/README.md#level-1-basic-), you haven't updated the project name.

**Comment by Soubeer** [comment]
> Hi @Soubeer, I can't keep telling you the same thing again and again. Yet, you still don't do things properly. [Here](https://github.com/Soubeer/Project-Guidance/blob/main/Machine%20Learning%20and%20Data%20Science/README.md#level-1-basic-), you haven't updated the project name.

mb Sir, updated and provided the link for the readme in basic with the main branch from ur repo...should activate the link once its approved...kindly check if there are any more problems.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
After 25 , 26 should be in right hand side. And this is the last time I'm saying if this time everything not correct I'll close the PR.

here 26

**Review by Kushal997-das — APPROVED** [review]
@Soubeer 👍🏻
